src/NzbDrone.Core/Notifications/Plex/PlexServerProxy.cs

```diff
@@ -170,26 +170,40 @@ private RestRequest GetPlexServerRequest(string resource, Method method, PlexSer
             var request = new RestRequest(resource, method);
             request.AddHeader("Accept", "application/json");
 
-            if (!settings.Username.IsNullOrWhiteSpace())
+            if (settings.Username.IsNotNullOrWhiteSpace())
             {
-                request.AddParameter("X-Plex-Token", GetAuthenticationToken(settings.Username, settings.Password), ParameterType.HttpHeader);
+                request.AddParameter("X-Plex-Token", GetAuthenticationToken(settings), ParameterType.HttpHeader);
             }
 
             return request;
         }
 
-        private string GetAuthenticationToken(string username, string password)
+        private string GetAuthenticationToken(PlexServerSettings settings)
         {
-            return _authCache.Get(username, () => Authenticate(username, password));
+            var token = _authCache.Get(settings.Username + settings.Password, () => Authenticate(settings));
+
+            if (token.IsNullOrWhiteSpace())
+            {
+                throw new PlexAuthenticationException("Invalid Token - Update your username and password");
+            }
+
+            return token;
         }
 
-        private void CheckForError(IRestResponse response)
+        private void CheckForError(IRestResponse response, PlexServerSettings settings)
         {
             _logger.Trace("Checking for error");
 
             if (response.StatusCode == HttpStatusCode.Unauthorized)
             {
-                throw new PlexException("Unauthorized");
+                if (settings.Username.IsNullOrWhiteSpace())
```

Comment:
We're not trying to authenticate here (we don't try to authenticate if the username hasn't been set), but we can still get an unauthorized response and this lets us tailor the message in that case.